htdocs/index.php

```diff
@@ -25,6 +25,7 @@
 // any authentication middleware, because that's done dynamically
 // based on the module router, depending on if the module is public.
 $middlewarechain = (new \LORIS\Middleware\ContentLength())
+    ->withMiddleware(new \LORIS\Middleware\HeadersGenerator())
```

Comment:
isn't this going to lose the content length header in the response?

Reply:
You are right. I change the order. But also consider your next comment